task_stages.go

```diff
@@ -15,8 +15,12 @@ type TaskStages interface {
 	// Read a task stage by ID
 	Read(ctx context.Context, taskStageID string, options *TaskStageReadOptions) (*TaskStage, error)
 
-	// List all task stages for a given rrun
+	// List all task stages for a given run
 	List(ctx context.Context, runID string, options *TaskStageListOptions) (*TaskStageList, error)
+
+	// **Note: This function is still in BETA and subject to change.**
+	// Override a task stage for a given run
+	Override(ctx context.Context, taskStageID string) (*TaskStage, error)
```

Comment:
I think this is missing the option to add a comment to the override. e.g. https://github.com/hashicorp/go-tfe/blob/main/run.go#L43

``` go
// TaskStageOverrideOptions represents the options for overriding a TaskStage.
type TaskStageOverrideOptions struct {
	// An optional explanation for why the stage was overridden
	Comment *string `json:"comment,omitempty"`
}
```

```suggestion
	Override(ctx context.Context, taskStageID string, options TaskStageOverrideOptions) (*TaskStage, error)
```